extensions/wikia/ArticleVideo/ArticleVideoController.class.php

```diff
@@ -1,16 +1,67 @@
 <?php
 
 class ArticleVideoController extends WikiaController {
-	public function index() {
-		$wg = F::app()->wg;
+	public function featured() {
+		global $wgTitle;
 
-		$wg->Out->addModules( 'ext.ArticleVideo' );
+		$wg = $this->getApp()->wg;
+		$title = $wgTitle->getPrefixedDBkey();
 
-		$title = RequestContext::getMain()->getTitle()->getPrefixedDBkey();
+		$enableArticleFeaturedVideo =
+			$wg->enableArticleFeaturedVideo &&
+			isset( $wg->articleVideoFeaturedVideos[$title] ) &&
+			$this->isFeaturedVideosValid( $wg->articleVideoFeaturedVideos[$title] );
 
-		// TODO: replace it with DS icon when it's ready (XW-2824)
-		$this->setVal( 'closeIconUrl',
-			$wg->extensionsPath . '/wikia/ArticleVideo/images/close.svg' );
-		$this->setVal( 'videoDetails', $wg->articleVideoFeaturedVideos[$title] );
+		if ( $enableArticleFeaturedVideo ) {
+			$wg->Out->addModules( 'ext.ArticleVideo' );
+
+			// TODO: replace it with DS icon when it's ready (XW-2824)
+			$this->setVal( 'closeIconUrl',
+				$wg->extensionsPath . '/wikia/ArticleVideo/images/close.svg' );
+			$this->setVal( 'videoDetails', $wg->articleVideoFeaturedVideos[$title] );
+		} else {
+			$this->skipRendering();
+		}
+	}
+
+	public function related() {
+		global $wgTitle;
+
+		$wg = $this->getApp()->wg;
+		$title = $wgTitle->getPrefixedDBkey();
+
+		$relatedVideo = null;
+		if ( $wg->enableArticleRelatedVideo &&
+		     isset( $wg->articleVideoRelatedVideos )
+		) {
+			$relatedVideo =
+				self::getRelatedVideoData( $wg->articleVideoRelatedVideos, $title );
+		}
+		$enableArticleRelatedVideo = $relatedVideo && $this->isRelatedVideosValid( $relatedVideo );
+
+		if ( $enableArticleRelatedVideo ) {
+			$this->setVal( 'relatedVideo',
+				self::getRelatedVideoData( $wg->articleVideoRelatedVideos, $title ) );
```

Comment:
Please stick to the [guidelines](https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md#multi-line-statements). So, this would be:
```
$this->setVal( 
  'relatedVideo',
  self::getRelatedVideoData( $wg->articleVideoRelatedVideos, $title )
);
```